src/cbdgen-framework.py

```diff
@@ -29,44 +29,24 @@
 tread = ""
 select_new_dataset = "N"
 NGEN = 1000
+# NGEN = options['NGEN']
 CXPB = 0.7
 MUTPB = 0.2
 INDPB = 0.05
 POP = 100
 
-n_instancias = options['samples']
-n_features = options['attributes']
-n_classes = options['classes']
-
-dataset = options['maker'][0]
-
-if(dataset == 1):
-    centers = int(options['maker'][1])
-    df = generate.blobs(n_instancias, centers, n_features)
-if (dataset == 2):
-    noise = options['maker'][1]
-    df = generate.moons(n_instancias, noise)
-if (dataset == 3):
-    noise = options['maker'][1]
-    df = generate.circles(n_instancias, noise)
-if (dataset == 4):
-    df = generate.classification(n_instancias, n_features, n_classes)
-if (dataset == 5):
-    n_labels = int(options['maker'][1])
-    df = generate.multilabel_classification(n_instancias, n_features, n_classes, n_labels)
+# TODO: Implement Generator of Instances in a minimal main()
```

Comment:
This needs to be adressed when [#20](https://github.com/SteffanoP/cbdgen-framework/issues/20) is fixed.

Reply:
Addressed to #25